bumpversion/__init__.py

```diff
@@ -340,7 +340,7 @@ class VersionConfig(object):
     Holds a complete representation of a version string
     """
 
-    def __init__(self, parse, serialize, search, replace, part_configs=None):
+    def __init__(self, parse, serialize, search, replace, part_configs={}):
```

Comment:
That's what code review is for, finding the oversights that are obvious if you're not the person coding.